admin/js/boldgrid-backup-admin-folder-exclude.js

```diff
@@ -19,7 +19,7 @@ BoldGrid.FolderExclude = function( $ ) {
 		exclusionList = null,
 		filteredList = [],
 		lang = BoldGridBackupAdminFolderExclude,
-		$container = $( '#folder_exclusion' ),
+		$container = $( 'div#folder_exclusion' ),
```

Comment:
We we need to have the selector this specific?

Reply:
Yes. In fact, the reason why we need to have teh div#folder_exclusion is because of the duplicate id's on the page. This is a result of the folder_exclusion element id's being present in the settings page, as well as in the Backup Now Modal. In order for them to both work, then the div#folder_exclusion must be present. This causes jQuery to treat the result as a list of elements, instead of picking the first one it finds.

Reply:
This is actually the solution suggested by Rafael